apps/docs/src/app/platform/component-docs/platform-forms/platform-textarea/platform-textarea-examples/platform-textarea-basic-example.component.html

```diff
@@ -0,0 +1,95 @@
+<main class="fd-page">
+    <div *ngIf="true" class="fd-page__content fd-has-background-color-neutral-2">
+        <fdp-form-group #fg1 [multiLayout]="true" [formGroup]="form" [object]="data">
+            <fdp-form-field
+                #ff1
+                [id]="'basicTextarea'"
+                label="Basic Textarea with Platform Forms"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+            >
+                <fdp-textarea [name]="'basicTextarea'" [formControl]="ff1.formControl"></fdp-textarea>
+            </fdp-form-field>
+
+            <fdp-form-field
+                #ff2
+                [id]="'readonlyDescription'"
+                label="Readonly Textarea"
+                [placeholder]="'Start entering detailed description'"
+                hint="This is tooltip help"
+                zone="zLeft"
+                rank="10"
+                hintPlacement="'left'"
+            >
+                <fdp-textarea [name]="'readonlyDescription'" [formControl]="ff2.formControl" [readonly]="true">
+                </fdp-textarea>
+            </fdp-form-field>
+
+            <!-- <fdp-form-field
+                #ff3
+                [id]="'disabledDescription'"
+                label="Disabled Textarea"
```

Comment:
unused code can be removed.

Reply:
This is an optional way of disabling through formControlName and by passing `disabled:true` in the FormControl initialization in .ts. Since both achieve the same purpose, I kept one commented, and this can be used for user's reference if they don't want to use `[disabled]="true"` method.

Reply:
@kavya-b any commented code won't work or won't be displayed. You can either uncomment and explain it or remove it

Reply:
ok.. removed